src/rpc/multipass.proto

```diff
@@ -172,15 +172,15 @@ message MountInfo {
 
 message InstanceStatus {
     enum Status {
-        RUNNING = 0;
-        STARTING = 1;
-        RESTARTING = 2;
-        STOPPED = 3;
-        DELETED = 4;
-        DELAYED_SHUTDOWN = 5;
-        SUSPENDING = 6;
-        SUSPENDED = 7;
-        UNKNOWN = 8;
+        UNKNOWN = 0;
```

Comment:
The order change also requires the client and daemon versions to match for the states to be interpreted coherently. I suppose that is alright, as long as this does not get into a bug release.

Reply:
Agreed this is slightly concerning, but since we typically tie the CLI closely to the server, it shouldn't be an issue. And we've not frozen our RPC yet.

But I do like this change as gRPC has a nasty habit of using the default initialiser in the case that the variable is not set. So having 0 be an error state is a good decision.

Reply:
Yeah, I'm not aware of any 3rd party multipass clients, so if anyone upgrades multipass, they'll get the new client at the same time.  Besides, as @gerboland points out, our RPC is not deemed stable yet, so we have the freedom to make changes like this :smile:

Reply:
Sure, I was mostly thinking aloud here... And then just pointing out this is a backward-incompatible change, so it means increasing the middle part of the semantic version.